src/lib.rs

```diff
@@ -1005,6 +1014,25 @@ mod tests {
 		assert_eq!(Mnemonic::from_entropy(&vec![b'x'; 36]), Err(Error::BadEntropyBitCount(288)));
 	}
 
+	#[test]
+	fn debug_does_not_leak_phrase() {
+		let m = Mnemonic::from_entropy(&[0u8; 16]).unwrap();
+
+		let dbg = format!("{:?}", m);
+
+		assert!(dbg.starts_with("Mnemonic {") || dbg.starts_with("Mnemonic(") || dbg.contains("Mnemonic"),
```

Comment:
nit: redundant cases: the last `contains` will match anything the previous cases match.